### DISCUSSION TITLE
Failing initial setup of project

### DISCUSSION THREAD

**Discussion by correaswebert** [stdlib-js/stdlib#2037] [Q&A]
I cloned the repository and ran am trying to install dependencies by running `make install` as per [stdlib/CONTRIBUTING](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md#step-2-initial-setup). I get the following error.

```
npm ERR! request to https://codeload.github.com/kgryte/tape/tar.gz/360c2fda548ed7a0120491a90196518a4b1dbaa7 failed, reason: self signed certificate in certificate chain
```

**Comment by kgryte** [comment]
@correaswebert Hmmm...I am not sure the reason for that error. We install our fork of `tape` from GitHub as a dev dep. So as long as `npm` supports installing from GitHub, it should work.

**Reply by kgryte** [reply]
Actually, I've never seen the `codeload` subdomain before. Here is the relevant line from our package.json:

https://github.com/stdlib-js/stdlib/blob/8204e8b47b11bfc12c12c659de368754a14fbd34/package.json#L240

**Reply by correaswebert** [reply]
Oh, gotta figure out a way to tackle this. Thanks :)
